modules/working-with-accelerator-profiles.adoc

```diff
@@ -6,11 +6,27 @@
 [role='_abstract']
 The accelerator profiles feature is currently available as a Technology Preview. For further information about the scope of Technology Preview status, and the associated support implications, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 
-To configure accelerators for your data scientists to use in {productname-short}, you must create an associated accelerator profile. An accelerator profile is a custom resource definition (CRD) on OpenShift that has an AcceleratorProfile resource, and defines the specification of the accelerator.  
+To configure accelerators for your data scientists to use in {productname-short}, you must create an associated accelerator profile. An accelerator profile is a custom resource definition (CRD) on OpenShift that has an AcceleratorProfile resource, and defines the specification of the accelerator. You can create and manage accelerator profiles by selecting *Settings* -> *Accelerator profiles* on the {productname-short} dashboard.
 
-For accelerators that are new to your deployment, you must manually configure an accelerator profile for each accelerator. If your deployment contained an accelerator before you upgrade, the associated accelerator profile preserves after the upgrade. You can manage the accelerators that appear to your data scientists by assigning specific accelerator profiles to your custom notebook images.
+For accelerators that are new to your deployment, you must manually configure an accelerator profile for each accelerator. If your deployment contained an accelerator before you upgrade, the associated accelerator profile preserves after the upgrade. You can manage the accelerators that appear to your data scientists by assigning specific accelerator profiles to your custom notebook images. This example shows the code for a Habana Gaudi 1 accelerator profile:
```

Comment:
- Could update 'contained' to 'contains' here, to stay in the one tense.
- 'Remains' might be more accurate than 'preserves' here, if we want to avoid passive language ('is preserved').